configs/sampler/pcdSampler.yaml

```diff
@@ -0,0 +1,4 @@
+# @package _group_
```

Comment:
This is the cfg file to create a `PCD` instance with specified init-parameters. In particular, the definition of the "batch_size" as init parameter.